### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by alexawesomecode**
### What does this Pull Request do?

This Pull request applies all the functionalities of the New York Times Project.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,473 @@
...
+nav img {
+  position: relative;
+  left: 2%;
+  width: 200px;
+  text-align: left;
+}
+
+.side-sect {
+  /* float: right; */
+  width: 4rem;
```
Comment (stratospherique) [inline style.css]: #### Remove commented code :mag:

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work Team :clap: . However, some changes needs to be made in order for us to approve your project. check :arrow_down: 

#### Page structure

##### Navigation bar

* [x] It has the NY Times image logo embedded in the center.
  - The image should be centered
* [x] The NY Times logo is a 
  - The image should be wrapped inside a link

##### Main Content

* [x] It contains centered text and images.
  - The telescope image is not centered.

##### Bottom part

* [x] It contains links to other articles with image previews on the leftmost side.
* [x] It contains links to more popular articles on the rightmost side.
  - Use `<a>` tag to make your links alive (i.e don't use just plain text and images, make them clickable)

#### Implementation details

##### Footer

* [x] It should have a fixed position (it sticks to the bottom part of the screen).
  - Add a fixed footer. You can clone the original site's bootom AD that sticks at the bottom of the viewport.

#### Implementation details

* [x] It uses [`<aside>` tag](https://www.w3schools.com/tags/tag_aside.asp) for `Editor's picks` section on the righmost side.
  - Identify this section with a noticeable heading.
* [x] **The Theory of Inflation** Should be just like the original site. :pray: grap the real image and align it properly.

#### General requirements: 

* [x] Wrong usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements. They should be "appropriately sized and positioned" (as described in Odin project description).
  - the footer cannot be outside of the `<body>` element. :no_entry_sign: 

* [x] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload.

* [x] The project has a descriptive README.md 

### That's it :relaxed: . When finished, kindly resubmit another code review request :pray:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,666 @@
...
+        <img
+          class="humberg"
+          src="https://cdn2.iconfinder.com/data/icons/music-player-icons-filled/41/Menu_Bar-512.png"
+          alt="menu"
+        />
+        <img
+          class="humberg"
+          src="http://www.pngmart.com/files/8/Search-Button-PNG-Image-Free-Download.png"
+          alt="menu"
+        />
```
Comment (whiz25) [inline index.html]: add text  (```space & cosmos```) with link.

Comment (whiz25) [reply index.html]: use ```font-awesome for the icons```. makes life easy compared to images.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good guys :+1: 
Your project looks awesome and meets most requirements. However, a few changes need to be made.
- [ ] your page is lacking the ```Editor's pick``` section. This is required so please add it with ```<aside>``` tags for the articles.
- [ ] look also at my comments and make those changes.

### Make the suggested changes and re-submit for another review.

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

**Review by Forison — COMMENTED** [review]
# Hi team 👋👋
You are not supposed to merge to master until you have had the go-ahead from a TSE to do so.😔
The project which does not follow the recommended git-flow would not be approved😔.
From what is see the previous TSE ask you to make some and submit for review I don't know why you merge after rectification😔
By the way, your project output looks awesome👌👌 , but I have a few changes we would like you to make 

* [ ] You should fix the *[errors]* *[warnings]* given to you by the[ validator ](https://validator.w3.org)when uploading the index file.🙂
* [ ] open a new pull request and submit for review
> make sure you follow git-flow this time

##### Guys, please fix the errors I have pointed out and submit your project for another review
feel free to contact me on [LinkedIn](https://www.linkedin.com/in/forison-addo-558201159/), [Twitter](https://twitter.com/ADDOBOAKYE1) or slack for further clarification if necessary. I would be happy to assist you.
Thank you 🙂
Happy coding

**Pull request merged**